packages/react-core/src/components/Modal/ModalBoxTitle.tsx

```diff
@@ -17,25 +32,50 @@ export const ModalBoxTitle: React.FunctionComponent<ModalBoxTitleProps> = ({
   className = '',
   id,
   title,
+  titleIconVariant,
+  titleLabel = '',
   ...props
 }: ModalBoxTitleProps) => {
   const [isTooltipVisible, setIsTooltipVisible] = React.useState(false);
   const h1 = React.useRef<HTMLHeadingElement>();
+  const label =
```

Comment:
Can you define `const isVariantIcon = isVariantIcon(titleIconVariant);` and then use `isVariantIcon` 3 places?

Reply:
You use the same name for both the boolean variable and the function here.
Did you have some other name in mind?

Reply:
Ah sorry, you can call it something else like `isTitleIconVariant`!

Reply:
When I define this const and reuse it, somehow typescript is complaining about the types.
 
```
packages/react-core/src/components/Modal/ModalBoxTitle.tsx:65:46 - error TS2538: Type 'FunctionComponent<any>' cannot be used as an index type.

65           {isTitleIconVariant ? variantIcons[titleIconVariant] : <CustomIcon />}
```

I did not research how to fix it a lot to be honest, but I don't know from the top of my head.